google/cloud/forseti/common/util/email/sendgrid_connector.py

```diff
@@ -36,15 +35,24 @@
 LOGGER = logger.get_logger(__name__)
 
 
-class EmailUtil(object):
-    """Utility for sending emails."""
+class SendgridConnector(base_email_connector.BaseEmailConnector):
+    """Utility for sending emails using Sendgrid API Key."""
 
-    def __init__(self, api_key):
+    def __init__(self, sender, recipient, kwargs):
         """Initialize the email util.
 
         Args:
-            api_key (str): The SendGrid api key to auth email service.
+            sender (str): The email sender.
+            recipient (str): The email recipient.
+            **kwargs (dict): A set of authentication attributes
+            corresponding to the selected connector.
         """
+        self.sender = sender
+        self.recipient = recipient
+        if kwargs.get('api_key'):
+            api_key = kwargs.get('api_key')
+        else:
```

Comment:
Make a comment on which one is for backward compatibility.

Reply:
Done